CoreDataAccessView/src/au/gov/asd/tac/constellation/views/dataaccess/panes/DataAccessPane.java

```diff
@@ -222,19 +222,17 @@ public DataAccessPane(DataAccessViewTopComponent topComponent) {
                 } 
             }
             // when no graph present, create new graph
-            if(graphId == null){
-                if (pluginSelected && selectedPluginsValid) {
-                    NewDefaultSchemaGraphAction graphAction = new NewDefaultSchemaGraphAction();
-                    graphAction.actionPerformed(null);
-                    while(GraphManager.getDefault().getActiveGraph() == null){
-                        // Wait and do nothing while graph is getting made
-                    }
-                    graphId = GraphManager.getDefault().getActiveGraph().getId();
-                    if (!graphState.containsKey(graphId)) {
-                        graphState.put(graphId, new GraphState());
-                    }
-                    currentGraphState = graphState.get(graphId);
+            if(graphId == null && pluginSelected && selectedPluginsValid){
```

Comment:
Is that the same as?

```
if(graphId == null && (pluginSelected && selectedPluginsValid)){
```

Reply:
Given they are all &&'s, it should be. I added braces to statement where I felt that distinction was needed (usually if it also involved ||'s). Otherwise (in the case of only &&'s) I opted to not include braces because they wouldn't make a difference to the logic

Reply:
Yep you are right. For some reason I thought the first was an `||`. Just ignore me!